app/src/components/main/grid/gridBody/gridRow/gridRow.jsx

```diff
@@ -82,13 +91,25 @@ export class GridRow extends Component {
   render() {
     const { columns, value, selectable, changeOnlyMobileLayout, rowClassMapper } = this.props;
     const customClasses = (rowClassMapper && rowClassMapper(value)) || {};
+
     return (
       <div
         className={cx('grid-row-wrapper', {
           selected: this.isItemSelected(),
           ...customClasses,
         })}
+        data-id={value.id}
+        ref={this.rowRef}
       >
+        <div className={cx('grid-row', 'highlight-block-wrapper')}>
+          <div
+            id="highlight-block"
```

Comment:
It shouldn't be unique per page, because we looking for this element in children of certain row with certain `id`.
Fixed - use another attribute - `datatype` .